modules/VtigerBackup/VtigerBackup.php

```diff
@@ -26,7 +26,12 @@
  * @author MAK
  */
 class VtigerBackup {
-	private $folderList = array('storage','test','user_privileges');
+	// 'storage' folder removed from backup, the CBX one is too large and will run into memory limits
```

Comment:
I can't accept this as copying storage files is fundamental. You now have an external backup method which will do big storage directories correctly.